src/models/tiles/geometry/jxg-board.ts

```diff
@@ -230,16 +231,10 @@ function createBoard(domElementId: string, properties?: JXGProperties) {
     // pan, or keyboard controls are not persisted to the model and so would be
     // more frustrating than helpful.
     // For accessibility, it would be very nice to have these work.
-    zoom: {
-      enabled: false,
-      wheel: false
-    },
-    pan: {
-      enabled: false
-    },
-    keyboard: {
-      enabled: false
-    }
+    zoom: { enabled: false },
```

Comment:
We're no longer disabling `wheel`?

Reply:
`enabled: false` should turn off all built-in zooming, including via mouse wheel.